packages/remix-dev/__tests__/fixtures/cloudflare/server.ts

```diff
@@ -4,5 +4,5 @@ import * as build from "@remix-run/dev/server-build";
 export const onRequest = createPagesFunctionHandler({
   build,
   getLoadContext: (context) => context.env,
-  mode: process.env.NODE_ENV,
+  mode: build.mode,
```

Comment:
Do we want to simplify `createPagesFunctionHandler` so we don't have to pass `mode` anymore and it gets automatically recognized?

Reply:
Conceptually, yes, but holding off on that since we are discussing better ways to configure dev-only options than to just lump them all under `NODE_ENV=development`